server/src/util.js

```diff
@@ -0,0 +1,61 @@
+const path = require('path');
+const fs = require('fs');
+
+/**
+ * Parses the input specification
+ * @param {object} file - The input file received as form-data
+ * @returns {object} The JSON for the input specification / the path to the input specification file
+ */
+function parseInputFile(file) {
+  const fileExt = path.parse(file.originalname).ext;
+  var origFile = Buffer.from(file.buffer).toString();
+
+  if(fileExt === '.json') {
+    origFile = JSON.parse(origFile);
+    return origFile;
+  }
+  else if(fileExt === '.yaml') {
+    const filePath = './input.yaml';
+    fs.writeFileSync(filePath, origFile);
+    return filePath;
+  }
+  else {
+    return null;
+  }
+}
+
+/**
+ * Sends response based on the requirement
+ * @param {object} result - The output file received from lucy's converter
+ * @param {object} file - The input file received as form-data
```

Comment:
This is only needed for input file extension, right?

The input file extension is only used to unlink the input file - this would need changes depending on another thread where we're discussing the need for saving the yaml file to disk